lb_content_resolver/formats/dbcleanup.py

```diff
@@ -30,83 +30,83 @@ def databaseCleanup(file):
     cur = dbc.cursor()
     rows = []
 
-    print "Checking tracks"
+    print("Checking tracks")
```

Comment:
I changed all prints into function calls, hope this is OK.

Reply:
Of course, I took this code from a python 2 codebase and clearly didn't clean everything up yet.

Reply:
Oh, dbcleanup -- I should've noted that I hadn't worked on this file at all yet. Partially because the indexing has bugs, making this script pointless currently.